image.yaml

```diff
@@ -31,7 +31,7 @@ modules:
           - name: jboss-eap-7-image
             git:
                 url: https://github.com/jboss-container-images/jboss-eap-7-image.git
-                ref: EAP_724_CR2
+                ref: EAP_724_1
```

Comment:
This will need to wait for the *EAP_725_\** tag yet.

Reply:
Ok